### PR TITLE
Add a "WASI proposal how-to" blog post

### PR DESCRIPTION
**Pull request by abrown**
This moves the second blog post of #10 into a separate commit for separate publication:
 - Implementing a WASI Proposal in Wasmtime: wasi-nn, [rendered](https://github.com/bytecodealliance/bytecodealliance.org/blob/04da5e09d68d7ce3c2800f84a71b99e84f131912/_posts/2020-12-05-implementing-wasi-nn-in-wasmtime.md)

### CODE REVIEW

File: _posts/2020-12-05-implementing-wasi-nn-in-wasmtime.md [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+
+Since OpenVINO&trade; does not have Rust bindings, I implemented both
+[`openvino-sys`](https://crates.io/crates/openvino-sys) and
+[`openvino`](https://crates.io/crates/openvino) and published the crates. Rust bindings (allowing
+calls from Rust to the backing implementation) are necessary for integration with Wasmtime; other
+Wasm engines will have different language requirements.
+
+Note: Using OpenVINO&trade; means that the models accepted by this implementation must be in
+OpenVINO&trade; IR format. This format uses two files for encoding an ML graph: a graph description
+XML file and a binary file with the weights. The OpenVINO&trade; IR is not the most comment encoding
```
Comment (linclark) [inline _posts/2020-12-05-implementing-wasi-nn-in-wasmtime.md]: I'm not sure whether this is a typo, or perhaps I just didn't parse the sentence correctly

```suggestion
XML file and a binary file with the weights. The OpenVINO&trade; IR is not the most common encoding
```

**Review by linclark — APPROVED** [review]
This looks great! It will be a great resource for others—thanks for putting it together!

I noticed one thing that I thought might be a typo, but other than that, no changes from me.

File: _posts/2020-12-05-implementing-wasi-nn-in-wasmtime.md (hunk 1 above)
Comment (abrown) [reply _posts/2020-12-05-implementing-wasi-nn-in-wasmtime.md]: Thanks, fixed!

**Comment by abrown** [comment]
This should be ready to merge; I changed the publication date to the 18th (Friday).

**Comment by sunfishcode** [comment]
Looks good to me too!

**Pull request merged**